frameworks/Python/uvicorn/uvicorn_conf.py

```diff
@@ -0,0 +1,13 @@
+import multiprocessing
+import os
+
+_is_travis = os.environ.get('TRAVIS') == 'true'
+
+workers = multiprocessing.cpu_count()
+if _is_travis:
+    workers = 2
+
+bind = "0.0.0.0:8080"
+keepalive = 120
+errorlog = '-'
+pidfile = 'uvicorn.pid'
```

Comment:
@tomchristie That would be fantastic if you could do that for all the frameworks! `/tmp` is preferable since it gets cleaned at the end of each framework test